### ISSUE TITLE
Standard donation amounts

### ISSUE THREAD

**Issue by Changaco** [liberapay/liberapay.com#975]
This issue is to discuss possible changes to standard donation amounts. For example I've heard concerns about the "Symbolic" amount being too low.

![screenshot of donation amount selector](https://user-images.githubusercontent.com/1581590/35932109-18294ab0-0c37-11e8-805f-842c16a9168b.png)

**Comment by Changaco** [comment]
Today approximately 16% of the donations on Liberapay are at 1 cent per week.

**Comment by jorgesumle** [comment]
I propose €0.05 per week and the same increase (500%) for the dolar.

**Comment by L1am0** [comment]
I would like the following steps:
0.1€ - symbolic
0.5€ - small
1.0€ - medium
5.0€ - Large
100.0€ - Maximum

Why this high (1000%) change: For me as donor, if I decide to support a project, I choose not by the money amount. I choose by the word next to it..."Oh I like this project, let's suport it with a 'small' amount". It is about me feeling good to give that to the project. 1€ or 2€ per Month do not change that.

As i see it (as competition to patrion): Liberapay should make it (in mid future) possible for indiviuals to life just of their projects, or to get an significant enough amount of money, so that they can focus on it for e.g. an entire workday per week. I don't see that coming with having 16% of all donations beeing just 1ct. So you would need 400 donors for a lunch :-1: 

Sure: 
Via the custom field, there should still be 0.01€ available & allowed!
And also the current donors stay at 0.01€ and don't get some sort of force upgraded to the new system.

**Comment by trebmuh** [comment]
What about giving the receiver/project the choice to select the different amounts for the different _words_ (symbolic/small/...) ? Then it would fit better each project/receiver scope and would give more freedom to every project.

From the UI, I'd suggest a defaulting template such as the current one, and a possibility to change the amount with a "change the value" button for the ones who want.

**Comment by Changaco** [comment]
Yes, allowing the profile owner to set custom amounts is one way we could go. The problem is that it could surprise/confuse donors who support multiple people/projects.

**Comment by trebmuh** [comment]
I'm not sure it would lead to a big confusion. People are used to choose between several options (this is how Patreons works IIRC). I agree it could surprise some people, especially the ones already owning a _receiver_ profile in LP because it's a change, then I'd suggest to leave a question mark or a link below the "Custom" item line which links to a page/paragraph where it's written something along this line:

> Liberapay is providing every profile owner the freedom to set up the 5 basic donation amounts (symbolic, small, medium, large, maximum) the potential patrons are proposed to choose from when they are on their way to make a donation.
>
> _Profile owners: bear in mind that it's **only** a proposition/advice you're giving to your patrons, they still can choose any donation amount they want!_

**Comment by yaxu** [comment]
I'm surprised 16% of donors chose 'symbolic'. None of my donors chose that so far.

€0.01 is practically nothing but on the other hand that's what 'symbolic' means. The value is not in the money but in the thought that someone wants to support my work, but (presumably) doesn't have the funds to do so. I currently get €14.32 per week which doesn't pay for much of my time but helps a bit, and I find really encouraging that 17 people want to support my free/open source projects. So in a way it's all symbolic for me at this point!

That said, €52 per year for a 'medium' donation seems like a decent amount to me. It could be a bit higher but I'm kind of fine with the way things are.

Basically I don't have strong feelings on this..

**Comment by philo351** [comment]
Having a profile owner set the amount their donors can give simplifies the donation process and prevents the "sticker shock" that some might have when they find themselves being billed several weeks worth of donations along with fees. It would be nice to be able to tell prospective donors (before they commit) that supporting our profile will be exactly $3.00/wk and they'll be billed at a rate of $18.60 every six weeks. 

So glad we're discussing this! :D

**Comment by cedeber** [comment]
Hello,

I come late to the discussion but I find these choices weird.

First, you have 5 choices with 4 between 0€ to 5€ a week and then the last, BAM, 100€! The amounts should grow with constancy.

Secondly, having the same choices for everybody can not work. There is a huge difference if I do something during my week-end's free time and if I want to have a donations-based salary with a full-time open source activity.
For instance, I find the symbolic useless. I prefer a friendly thank-you on social networks than a half euro per year form an anonymous. But it's my personal point of view.

At least, if you don't want to let receivers choose the amounts they want, give us the choice to only show the custom option. In a psycho-business-logic it's also better to not give people choices and let them free to judge how much they want to spend. On average you receive more money then expected.

**Comment by Mikaela** [comment]
> The value is not in the money but in the thought that someone wants to support my work, but (presumably) doesn't have the funds to do so.

This is my situation, I am unemployed job seeker and I have to think of myself first and how will my situation stay stable. I also see symbolic as a reminder on the giving page that I wish to donate to those projects more when I can instead of having to keep a separate list (I hope I am making any sense).

* * * *

I actually came here from Gitter as currently the symbolic option on paying page looks strange and stops me from donating as I don't know which option I should pick.

>> Hi, is there a known issue about all payment options saying "not recommended"?
https://github.com/liberapay/salon/issues/304#issuecomment-490644735

> I don't think we have an open issue specifically about this problem. The most relevant one I can think of is liberapay/liberapay.com#975 because doubling the suggested Symbolic donation amount would indirectly solve the problem for most donors.

![Screenshot from 2019-05-08 23-44-50](https://user-images.githubusercontent.com/831184/57409658-5f76b300-71d8-11e9-9254-036779bca036.png)

I think I will wait for this issue to get resolved first before I donate and accidentally put more money to fees than the project I wish to support. My other donations are due in one month.

**Comment by CanIGetaPR** [comment]
In my opinion, symbolic is a way to show that your project is of interest to someone, Perhaps it is comforting for future patrons to see that the project already has many patrons (even if some are just symbolic).

It may increase the chance of the project getting new patrons when the page is new.

**Comment by brauliobo** [comment]
customizing the donations options is a basic need to adapt donations to each project, currency and receiver...

I've tried to customize the custom option via parameters and it didn't work, how to do it? What I've tried: https://en.liberapay.com/brauliobo/donate?amount=50&currency=usd&period=monthly&selected_amount=custom

**Comment by Changaco** [comment]
This isn't what this issue is about, but the reason your link doesn't work is that the currency code should be in uppercase: https://liberapay.com/brauliobo/donate?amount=50&currency=USD&period=monthly

**Comment by boehs** [comment]
|Weekly|Monthly|Yearly|
|------|-------|------|
|0.01  |0.04   |0.52  |
|0.05  |0.2    |2.6   |
|**0.1**   |0.4    |5.2   |
|**0.25**  |1      |13    |
|**0.5**   |2      |26    |
|**1**     |4      |52    |
|**2**     |8      |104   |
|**3**     |12     |156   |
|5     |20     |260   |
|10    |40     |520   |

I highlighted the ones I like the most
